src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 12px;
+  height: 12px;
+  padding: 0;
+  margin: 2px 4px;
+  background-image: url("images/star.svg");
+  background-size: cover;
+  transform: translateX(-2px);
+}
```

1. no needs to reset padding;
2. transform is redundant
3. use one side margin with `:not(:last-child)`
4. perhaps you'll need to add background-repeat: no-repeat;
```suggestion
.stars__star {
  width: 12px;
  height: 12px;
  background-image: url("images/star.svg");
  background-size: cover;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```